### PR TITLE
REFINE: improve the README documentation for setting up a KVM based t…

### PR DESCRIPTION
**Pull request by yejianquan**
### Description of PR

Summary:
Refine the README documentation for setting up a KVM based testbed.

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
Refine the README documentation for setting up a KVM based testbed.
#### How did you do it?

1.  Split Ubuntu 18.04 and 20.04 into to two individual optional ways in 'Prepare testbed host' step, otherwise if user followed 20.04 actions in 18.04 Ubuntu host, it may cause hard-to-track issues in 'add-topo' step. Current suggestion is: Python2 for Ubuntu 18.04, Python3 for Ubuntu 20.04, and don't let host installs Python2 and Python3 at one time.
2.  Remove the leading '$' for shell commands, reader will be able to copy them more friendly.
3.  Refine some statement and details to help totally new approacher understand easilier.

#### How did you verify/test it?

I just followed the documentation to created a KVM testbed successfully.

### CODE REVIEW

File: docs/testbed/README.testbed.VsSetup.md [hunk 1]
```diff
@@ -4,29 +4,36 @@ This document describes the steps to setup a virtual switch based testbed, deplo
...
-```
-ln -sf /usr/bin/python3 /usr/bin/python
-```
-
-4. Run the host setup script to install required packages and initialize the management bridge network
+2. Prepare your environment based on different Ubuntu version, make sure that python and pip are installed
+   1. Option : If your host is **Ubuntu 20.04**
+        
+        ```
+        apt install python3 python3-pip
```
Comment (wangxin) [inline docs/testbed/README.testbed.VsSetup.md:15]: sudo apt install python3 python3-pip

File: docs/testbed/README.testbed.VsSetup.md [hunk 2]
```diff
@@ -4,29 +4,36 @@ This document describes the steps to setup a virtual switch based testbed, deplo
...
+        ```
+        If the server was upgraded from Ubuntu 18.04, check the default python version using command `python --version`. If the default python version is still 2.x, replace it with python3 using symbolic link:
+        ```
+        ln -sf /usr/bin/python3 /usr/bin/python
```
Comment (wangxin) [inline docs/testbed/README.testbed.VsSetup.md:19]: sudo ln -sf /usr/bin/python3 /usr/bin/python

File: docs/testbed/README.testbed.VsSetup.md [hunk 3]
```diff
@@ -4,29 +4,36 @@ This document describes the steps to setup a virtual switch based testbed, deplo
...
+        ```
+   2. Option : If your host is **Ubuntu 18.04**
+        ```
+        apt install python python-pip
+        # v0.3.12b Jinja2 is required, lower version may cause uncompatible issue
+        pip install j2cli==0.3.12b0
```
Comment (wangxin) [inline docs/testbed/README.testbed.VsSetup.md:25]: I think prepending `sudo` to the command is required.

File: docs/testbed/README.testbed.VsSetup.md (hunk 1 above)
Comment (yejianquan) [reply docs/testbed/README.testbed.VsSetup.md]: fixed

File: docs/testbed/README.testbed.VsSetup.md (hunk 2 above)
Comment (yejianquan) [reply docs/testbed/README.testbed.VsSetup.md]: fixed

File: docs/testbed/README.testbed.VsSetup.md (hunk 3 above)
Comment (yejianquan) [reply docs/testbed/README.testbed.VsSetup.md:25]: added sudo for apt instll python python-pip
and pip install j2cli==0.3.12b0 doesn't need a sudo

**Review by wangxin — APPROVED** [review]

**Pull request merged**